README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
+- Recursion: Recursion is when you have a method that calls itself. All recursive methods can be written iteratively.
+- Recursive Case: The initial case where there isn't a known number.
+- Base Case: When you finally get to the point where you can input a known number and "work backwards" to get the answer to a recursive problem.
```

Comment:
This is true.

You could also say a base case is the case which stops the recursion by not calling the method again.